### ISSUE TITLE
Support for duotone icons

### ISSUE THREAD

**Issue by securit** [FortAwesome/vue-fontawesome#186]
Is support for FontAwesome Duotone icons on the development path

And if so, is this something sooner or later?

**Comment by hc-codersatlas** [comment]
The readme on the front page says duetone is in pre-release, but is that still so? I thought duotones have been released. Would like to know too.

**Comment by Livijn** [comment]
Just use `npm install @fortawesome/pro-duotone-svg-icons` and import them as usual.

**Comment by securit** [comment]
And how do you apply the different tones ?

**Comment by Livijn** [comment]
I don't think you can choose the secondary tone. It uses the same color but with about 50% opacity.

**Comment by marnitzmalan** [comment]
@Livijn I've installed duotone with npm (npm install @fortawesome/pro-duotone-svg-icons).. 
Next I want to import it in my scss file (@import "~@fortawesome/fontawesome-pro/scss/duo")
But there is no scss file created  for the due tone in the node vendor folder as there is for the rest of the variations.....(node_modules/@fortawesome/fontawesome-pro/scss/duo.scss)

**Comment by spockNinja** [comment]
I was able to utilize the `style` property in order to set both colors:

```
<font-awesome-icon :icon="['fad', 'rocket']" :style="{'--fa-primary-color': 'blue', '--fa-secondary-color': 'purple'}"/>
```

**Comment by Teyler7** [comment]
I have no idea if anyone else can relate, I am having this strange issue where the '--fa-primary-color' gets set, but the other color does not appear at all. I checked the computed css properties and there is nothing from it's parents that fa inheriting. I used @spockNinja 's code as a raw example to try and display two colors on different backgrounds (First image shows fas, second shows fad). Any thoughts?
![Screenshot (42)](https://user-images.githubusercontent.com/11841273/63188732-66a32d80-c01f-11e9-9aa0-8f928d248b67.png)
![Screenshot (43)](https://user-images.githubusercontent.com/11841273/63188733-66a32d80-c01f-11e9-83fd-ad7bdc2f6c11.png)
```html
<font-awesome-icon :icon="['fad', 'rocket']" :style="{'--fa-primary-color': 'red !important', '--fa-secondary-color': 'purple !important'}"/>
```
